backend/migrating/v2/management/commands/migrate_to_v2.py

```diff
@@ -202,6 +202,43 @@ def _prepare_row_and_migrate_relations(
         row = [json.dumps(value) if isinstance(value, dict) else value for value in row]
         return tuple(row)
 
+    def _convert_lists_to_json(
+        self,
+        row: tuple[Any, ...],
+        key: str,
+        column_names: list[str],
+    ) -> tuple[Any, ...]:
+        """Convert specified field in the row to JSON format if it is a
+        list."""
+        if key not in column_names:
+            return row
+
+        index = column_names.index(key)  # Index of the key
+        row_as_list = list(row)
+        if isinstance(row[index], list):
+            row_as_list[index] = json.dumps(row_as_list[index])
+            return tuple(row_as_list)
+        row = [
+            json.dumps(value) if isinstance(value, dict) else value
+            for value in row_as_list
+        ]
+        return row
+
+    def _process_type_transfer(
```

Comment:
@muhammad-ali-e why is this done? Can it be named better / add some docstrings?
We won't do anything if there's no `list` here